pages/index.js

```diff
@@ -17,6 +21,16 @@ export default function Index() {
     return false;
   }, [countySearchRef]);
 
+  const goToCountyIfNotNull = (counties) => {
+    if (counties && counties.length > 0) {
+      router.push(`counties/${counties[0].replace(" ", "_")}`);
```

Comment:
TODO before merging: Hitting the button without typing anything is broken. Need to push up an additional null check commit